nuitka/build/SconsCompilerSettings.py

```diff
@@ -628,6 +628,8 @@ def setupCCompiler(env, lto_mode, pgo_mode, job_count, onefile_compile):
             # compilation more memory hungry, but also faster.
             env.Append(CCFLAGS="-pipe")
 
+        # gcc complier fcf_protection option
```

Comment:
Also unclear if clang has it, and if there is maybe a GCC version limitation, although I think I am about to drop very old gcc support, but it might have to be made conditional on that.